pkg/apis/core/v1alpha1/filewatch_types.go

```diff
@@ -119,11 +119,13 @@ func (in *FileWatchList) GetListMeta() *metav1.ListMeta {
 
 // FileWatchStatus defines the observed state of FileWatch
 type FileWatchStatus struct {
+	// MonitorStartTime is the timestamp of when filesystem monitor was started.
+	MonitorStartTime metav1.MicroTime `json:"monitorStartTime,omitempty"`
 	// LastEventTime is the timestamp of the most recent file event. It is nil if no events have been seen yet.
```

Comment:
nil -> zero? (now that this isn't a pointer?)

Reply:
Ah, yep! FWIW the reason I changed these was from what I've seen, apiserver expects anything w/o `omitempty` to be **required** (even if it's a pointer type - assumption I guess is you're using pointer for perf reasons rather than nullability reasons?) so it doesn't really get us anything other than more annoying ergonomics on the Go side, and since we are required to use `omitempty`, the JSON side is the same either way